src/dlgConnectionProfiles.cpp

```diff
@@ -628,12 +634,19 @@ void dlgConnectionProfiles::slot_deleteProfile()
 
     QString profile = profiles_tree_widget->currentItem()->data(csmNameRole).toString();
 
+    QDir profileDirContents(mudlet::getMudletPath(mudlet::profileXmlFilesPath, profile));
```

Comment:
:thinking: How quickly will the autosave code kick in to put something in that directory?

... ah, given that copying a profile is gonna stuff things in there isn't it.

Reply:
I am not sure what do you mean, can you rephase?

Reply:
I was just wondering at what point in the selection of a predefined MUD profile does ***anything*** get written out to the corresponding directory in the file system so that it starts to *exist* as far as the OS's filesystem goes - the current methodology of storing profile details as discrete files will start to do that as the user moves around the fields on the right side of the connect profiles dialogue won't it...

... and so I was also wondering whether the auto saving stuff would be starting to act to populate the `./current/` directory for the profile - however I realise now that that is **not going to happen** as that can only occur after the load or connect buttons are hit and not whilst the user is navigating around this class's code!

Reply:
That's right.